src/env.d.ts

```diff
@@ -2,3 +2,7 @@ interface ImportMetaEnv {
     VITE_API_URL: string;
     VITE_THEMES_URL: string;
 }
+
+interface ImportMeta {
+    env: ImportMetaEnv
+}
```

Comment:
Ah, that explains a lot, thanks for fixing this since it's been bugging me a lot.